### ISSUE TITLE
outputs 500 results despite changing config

### ISSUE THREAD

**Issue by peiyaoli2** [twitterdev/search-tweets-python#33]
Hi! I have set the max-results inside my config file to be something other than 500, but the output always has 500 results no matter how I change the config file, even if max-results is smaller than 500. I wonder if there is anything I have done wrong in this case.

**Comment by binaryaaron** [comment]
@peiyaoli2, thanks for the notice.

Can you share:
- what your config file looks like (without keys, bearer tokens, or enterprise auth info)
- what version of the python library you're using
- how you are using the library (command-line application or in a program)
- the query you are running

Thank you!

**Comment by peiyaoli2** [comment]
Hi - I'm using python 3, and I'm using the library as a command-line application.  This is how my config.yaml looks like:

```yaml
search_rules:
    from-date: 2018-05-01
    to-date: 2018-05-24
    pt-rule: illinois2018 lang:en

search_params:
    results-per-call: 100
    max-results: 10000

output_params:
    save_file: True
    filename_prefix: test_illinois2018
    results_per_file: 10000000
```

and this is how my command-line looks like:
`C:\ProgramData\Anaconda3\Scripts>python search_tweets.py --credential-file twitt
er_keys.yaml --credential-file-key search_tweets_api --env-overwrite False --con
fig-file config.yaml --no-print-stream`

I actually wasn't sure about the env-overwrite flag, therefore I tried both True and False, but the results are the same.

**Comment by binaryaaron** [comment]
@peiyaoli2 - thank you; I think you've discovered a bug. I believe some of the parameters in the companion config file are being overwritten by default values in the command-line application. 

Until I can fix the underlying problem, specifying the parameters `--results-per-call` and `--max-results` on the command line should suffice. 

Thank you for reporting this!

**Comment by binaryaaron** [comment]
closed by #34.
